src/login.js

```diff
@@ -0,0 +1,65 @@
+import React, { useState} from "react";
+import Form from "react-bootstrap/Form";
+import Button from "react-bootstrap/Button";
+import {createPatient} from './firebase'
+import { useHistory } from 'react-router-dom';
+import { jsPsych } from 'jspsych-react'
+
+ function Login() {
+    const dateTimestamp = Date.now()
+    const curDate = new Date(dateTimestamp)
+    const [patientId, setPatient] = useState("");
+    const [studyId, setStudy] = useState("")
+    const [startDate] = useState(curDate.toString())
+    const [timestamp] = useState(dateTimestamp.toString())
+    const history = useHistory();
+
+    function validateForm() {
+        return patientId.length > 0 && studyId.length > 0;
+      }
+
+    function handleSubmit(e) {
+        e.preventDefault();
+        const logPatient = async() =>{
+            const loggedIn = await createPatient(patientId, studyId, startDate, timestamp)
```

Comment:
I think `loggedIn` should come form `useState` as well and remove the `LOGGEDIN`variable in `App.js`